src/containers/Dashboard.js

```diff
@@ -1,42 +1,231 @@
 import React from "react";
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
 import produce from "immer";
+import * as R from "ramda";
 
 import apiClient from "../lib/apiClient";
+import { jwtDecode } from "../lib/localStorage";
+
+// Components
+import DashboardApiKeys from "./DashboardApiKeys";
+import DashboardStats from "./DashboardStats";
+import DashboardUserSettings from "./DashboardUserSettings";
+
+// Styles
+import "../styles/Dashboard.css";
+import "../styles/Buttons.css";
 
 export default class Dashboard extends React.Component {
   constructor(props) {
     super(props);
 
     this.state = {
-      logoutError: ""
+      firstName: "",
+      lastName: "",
+      email: "",
+      userId: null,
+      apiKeys: [],
+      isLoading: true,
+      error: ""
     };
   }
 
+  componentDidMount() {
```

Comment:
Perhaps I'm missing something, but why does isLoading need to be set to true inside `componentDidMount` when the default state has it set to true?

Reply:
GOOD POINT